### DISCUSSION TITLE
FormatConvertedBitmap is missing

### DISCUSSION THREAD

**Discussion by pekspro** [dotnet/maui#4279] [Ideas]
In Xamarin you could use `FormatConvertedBitmap` to transform an image to greyscale (here is [an example](https://stackoverflow.com/questions/48848510/transform-a-colored-image-to-a-grayscale-image-using-xaml)). But in MAUI preview 12 this doesn't seem to be available. Is this something that will be added?

**Comment by hartez** [comment]
The example you linked to is for WPF, not Xamarin. As far as I know, Xamarin has never had a FormatConvertedBitmap method. And I don't believe we have anything like that on the roadmap. 
However, you might be able to accomplish this with [SkiaSharp](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/effects/image-filters) or [FFImageLoading](https://devlinduldulao.pro/how-to-use-ffimageloading/).

**Reply by pekspro** [reply]
Thanks. I checked both projects you mention but it looks like none these supports MAUI yet. But I have no rush, I could wait :-)  [ImageSharp](https://github.com/SixLabors/ImageSharp) might be another option.
